presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,27 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix queries failing with ``EXCEEDED_LOCAL_MEMORY_LIMIT`` error due to incorrect memory tracking while reading spilled data.
+* Fix deadlock for queries with ``JOIN`` and ``LIMIT`` with spilling enabled.
+* Fix query failures due to expressions with multiple ``IS_NULL`` on the same variable.
```

Comment:
The way it's written, it's as if `IS_NULL` is a function.  Let's remove the underscore and make it lower case to make it clearer. 

```suggestion
* Fix query failures due to expressions with multiple ``is null`` checks on the same variable.
```

Reply:
done